dev-docs/agents/coding-guidelines.md

```
@@ -1,233 +1,230 @@
+- Write docstrings in Google style
+- Write imports in the correct order
+- Keep error handling concise
+- Split functions when complexity gets too high
+
+---
+
+## 7.2 Recommended prompt for Codex output
+
+Paste the following for stable code generation:
+
+```
```

markdownlint-cli2 (0.21.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)